concourse/pipelines/destroy-microbosh.yml

```diff
@@ -98,6 +98,23 @@ jobs:
           - get: bosh-secrets
           - get: bosh-init-state
           - get: bosh-manifest
+      - task: bosh-cleanup
+        config:
+          platform: linux
+          image: docker:///governmentpaas/bosh-cli
+          inputs:
+            - name: paas-cf
+            - name: bosh-secrets
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/bosh_login.sh bosh-secrets/bosh-secrets.yml
+
+                # Cleanup ensures any orphaned disks are deleted before bosh itself is deleted
```

Comment:
> That's good, because it's stuff that we also want to happen.

In this instance, there's no need to clean the releases and stemcells up because we're about to delete the machine. It's just the volumes we care about. I had initially written a script to do just that, but it did them in serial, so it was actually quicker to do the full cleanup (and is one less script to maintain).